docs/topics/request-response.rst

```diff
@@ -316,6 +317,13 @@ download_timeout
 The amount of time (in secs) that the downloader will wait before timing out.
 See also: :setting:`DOWNLOAD_TIMEOUT`.
 
+.. reqmeta:: download_latency
+
+download_latency
+----------------
+
+The amount of time spent to fetch the response, since the request has been
+started.
```

Comment:
what about
> since the request has been started, i.e. HTTP message sent over the network.

(as opposed to when the Request is yielded by a callback or queued by the scheduler)